### PR TITLE
Review tribute page coding

### PR DESCRIPTION
**Pull request by kwambiee**
To review the tribute page:
1. I created a new repository and opened a branch
2. I copied the HTML and style sheet code to the new repository
3. I pushed the code to the remote repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# Review-Tribute-Page
\ No newline at end of file
+# Tribute_page
```
Comment (kwambiee) [inline README.md:1]: Let your README.md file be descriptive. Use this template (https://github.com/microverseinc/readme-template) to write accurate README.md

File: index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    
+    <main id="main">
```
Comment (kwambiee) [inline index.html:13]: Optional changes: There is a good use of semantic tags. Also note, it is important to create parent elements and child elements with div for easy styling.

File: tribute2.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=<device-width>, initial-scale=1.0">
+    <title>Tribute to Elizabeth</title>
+</head>
+<body>
+    <p>She was a mum of four children,two daughters and two sons and a grandmother to eight children. She was a loving wife to Mr Ezekiel Ochukwu.</p>
```
Comment (kwambiee) [inline tribute2.html:10]: Use the main tag. Unique content is usually placed inside the main tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (kwambiee) [inline index.html:11]: set up linters to check for styling and syntax errors.

**Review by kwambiee — COMMENTED** [review]
Changes required.

Please check the comments below and make the necessary changes.
Submit for review once you are done.

**Review by vic778 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @kwambiee , 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**